lib/contextual_logger/logger_with_context.rb

```diff
@@ -36,6 +38,27 @@ def write_entry_to_log(severity, timestamp, progname, message, context:)
       @logger.write_entry_to_log(severity, timestamp, progname, message, context: merged_context)
     end
 
+    def normalize_context(context)
```

Comment:
These methods should be `private` so that it's not part of the public contract, which will impact the way that you're testing this contract.